src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1 hidden>Search bar airbnb</h1>
+    <!-- Search Bar Big -->
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      data-qa="big"
+    >
+      <!-- Test setup <form action="" method="get" role="search" data-qa="big"> -->
+      <label class="bar bar-big" >
+        <img 
+          class="icon icon-big" 
+          src="images/Search.svg" 
+          alt=" "
+        >
+        <input 
+          type="search" 
+          name="q" 
+          class="field field-big" 
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <!-- Search Bar Small -->
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      data-qa="small"
+    >
+      <!-- Test setup <form action="" method="get" role="search" data-qa="small"> -->
+      <label class="bar bar-small">
+        <img 
+          class="icon icon-small" 
+          src="images/Search.svg" 
+          alt=" "
```

Comment:
Add a meaningful alt text for the search icon, e.g., alt="search icon".

Reply:
same here